test/isFilled.test.ts

```diff
@@ -1,198 +1,226 @@
+	expect(prismicH.isFilled.color(undefined)).toBe(false);
+	expect(
+		prismicH.isFilled.color(prismicM.value.color({ seed: ctx.meta.name })),
+	).toBe(true);
 });
 
-test("content relationship", (t) => {
-	t.false(prismicH.isFilled.contentRelationship(null));
-	t.false(prismicH.isFilled.contentRelationship(undefined));
-	t.false(
+it("content relationship", (ctx) => {
+	expect(prismicH.isFilled.contentRelationship(null)).toBe(false);
+	expect(prismicH.isFilled.contentRelationship(undefined)).toBe(false);
+	expect(
 		prismicH.isFilled.contentRelationship(
 			prismicM.value.contentRelationship({
-				seed: t.title,
+				seed: ctx.meta.name,
 				state: "empty",
 			}),
 		),
-	);
-	t.true(
+	).toBe(false);
+	expect(
 		prismicH.isFilled.contentRelationship(
 			prismicM.value.contentRelationship({
-				seed: t.title,
+				seed: ctx.meta.name,
 				state: "filled",
 			}),
 		),
-	);
+	).toBe(true);
 });
 
-test("date", (t) => {
-	t.false(prismicH.isFilled.date(null));
-	t.false(prismicH.isFilled.date(undefined));
-	t.true(prismicH.isFilled.date(prismicM.value.date({ seed: t.title })));
+it("date", (ctx) => {
+	expect(prismicH.isFilled.date(null)).toBe(false);
+	expect(prismicH.isFilled.date(undefined)).toBe(false);
+	expect(
+		prismicH.isFilled.date(prismicM.value.date({ seed: ctx.meta.name })),
+	).toBe(true);
 });
 
-test("embed", (t) => {
-	t.false(prismicH.isFilled.embed(null));
-	t.false(prismicH.isFilled.embed(undefined));
-	t.false(prismicH.isFilled.embed({}));
-	t.true(prismicH.isFilled.embed(prismicM.value.embed({ seed: t.title })));
+it("embed", (ctx) => {
+	expect(prismicH.isFilled.embed(null)).toBe(false);
+	expect(prismicH.isFilled.embed(undefined)).toBe(false);
+	expect(prismicH.isFilled.embed({})).toBe(false);
+	expect(
+		prismicH.isFilled.embed(prismicM.value.embed({ seed: ctx.meta.name })),
+	).toBe(true);
 });
 
-test("geopoint", (t) => {
-	t.false(prismicH.isFilled.geoPoint(null));
-	t.false(prismicH.isFilled.geoPoint(undefined));
-	t.false(prismicH.isFilled.geoPoint({}));
-	t.true(
-		prismicH.isFilled.geoPoint(prismicM.value.geoPoint({ seed: t.title })),
-	);
+it("geopoint", (ctx) => {
+	expect(prismicH.isFilled.geoPoint(null)).toBe(false);
+	expect(prismicH.isFilled.geoPoint(undefined)).toBe(false);
+	expect(prismicH.isFilled.geoPoint({})).toBe(false);
+	expect(
+		prismicH.isFilled.geoPoint(
+			prismicM.value.geoPoint({ seed: ctx.meta.name }),
+		),
+	).toBe(true);
 });
 
-test("group", (t) => {
-	t.false(prismicH.isFilled.group(null));
-	t.false(prismicH.isFilled.group(undefined));
-	t.false(prismicH.isFilled.group([]));
-	t.true(prismicH.isFilled.group(prismicM.value.group({ seed: t.title })));
+it("group", (ctx) => {
+	expect(prismicH.isFilled.group(null)).toBe(false);
+	expect(prismicH.isFilled.group(undefined)).toBe(false);
+	expect(prismicH.isFilled.group([])).toBe(false);
+	expect(
+		prismicH.isFilled.group(
+			prismicM.value.group({ seed: ctx.meta.name }) as prismicT.GroupField,
```

Comment:
Unrelated, but for SliceZones and group fields I had to force types like this for some obscure reasons 🤔 My wild guess would be that `@prismicio/mock` is out of sync but maybe there's a bigger issue under that, didn't have time to investigate properly.

Reply:
It looks like `@prismicio/mock`'s type doesn't return the property type for generated group or Slice Zone values that do not have an explicit model. It doesn't default a field value to "AnyRegularField" (at least not to something compatible with it).

i.e.:
```
ModelValueMap<Record<string, CustomTypeModelFieldForGroup>> !== Record<string, AnyRegularField>
```

It looks complicated, so we'll need to revisit at another time.